lib/jsonapi/view.ex

```diff
@@ -126,7 +126,9 @@ defmodule JSONAPI.View do
 
       def fields, do: raise("Need to implement fields/0")
 
-      def hidden, do: []
+      def hidden(), do: []
+
+      def hidden(data), do: hidden()
```

Comment:
Yep. Good catch. Looking at this again i wanted to hear your thoughts on passing just `data` vs passing `conn` as well. I can’t think of a scenario where I would need the conn, but I wondered whether it would make sense to keep it consistent with other functions in this module that take both conn and data.

Reply:
I don't like to pollute everything in the view with conn's, makes testing a bit harder and doesn't provide a ton of value outside of what our QueryParser already makes available.